Sources/LanguageServerProtocol/ServerCapabilities.swift

```diff
@@ -54,6 +54,9 @@ public struct ServerCapabilities: Codable, Hashable {
   /// The server provides workspace symbol support.
   public var workspaceSymbolProvider: Bool?
 
+  /// Whether the server provides "textDocument/implementation".
+  public var implementationProvider: Bool?
```

Comment:
Nitpick: could you move this up to right after `definitionProvider` and  do the same in the initializer?

Reply:
done